docs/001_develop/03_client-capabilities/008_interaction/scheduler.mdx

```diff
@@ -0,0 +1,259 @@
+
+Once configured, the component automatically generates a [Quartz-compatible cron expression](https://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html), handles validation and parsing, and emits update events when the schedule changes. This makes it easy to integrate with backend systems that rely on cron-based scheduling, and flexible enough to embed in various applications and frameworks.
+
+
+**Use cases:**
+* Automating database backups
+* Scheduling recurring data processing jobs
+* Sending periodic notifications or emails
+* Generating reports at regular intervals
+
+:::info
+ For a breakdown of how cron expressions work and the library used on the backend, see [Cron Expression Format (Quartz)](https://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html) or [`Cron expression section`](#cron-expression).
+:::
+
+## Example
+Lets look at a simple example of component in action. For example if you want the task to be executed on every Monday and Wednesday at 10:30 AM, you can select the `Weekly` recurrence type, choose `Monday` and `Wednesday` from the list of days, set the time to `10:30 AM`, and the component will generate the corresponding cron expression. The generated cron expression is displayed in the output field.
+You can play around with different recurrence types, each offering its own set of input options, and see how the cron expression updates in real time based on your selections.
+
+<CronSchedulerDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+      <rapid-scheduler-cron-builder></rapid-scheduler-cron-builder>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html<MyElement>`
+        <rapid-scheduler-cron-builder
+          :cron=${sync((x) => x.cronExpression)}
+          @change=${(x, c) => x.handleChange(c.event)}
+        ></rapid-scheduler-cron-builder>
+        <p>Cron: {(x) => x.cronExpression}</p>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      @observable cronExpression = '0 30 10 ? * 2,4';
+      
+      handleChange(event) {
+        console.log(event.target.value);
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+      <rapid-scheduler-cron-builder></rapid-scheduler-cron-builder>
+    ```
+
+    Usage
+    ```tsx
+    export function MyComponent() {
+      
+      const schedulerRef = useRef(null);
+      const [cron, setCron] = useState('0 30 10 ? * 2,4');
+
+      useEffect(() => {
+        if (schedulerRef.current) {
+          schedulerRef.current.cron = cron;
+        }
+      }, [cron]);
+
+      return (
+        <rapid-scheduler-cron-builder
+          ref={schedulerRef} 
+          cron={cron} 
```

Comment:
```suggestion
```

Glad to hear the useRef helped, but now I think you don' need to set the property on the JSX inline at all. Test that out and update the example and the actual code too